citadel/install_ubuntu.md

```diff
@@ -12,7 +12,7 @@ sudo apt-get install ignition-citadel
 
 All libraries should be ready to use and the `ign gazebo` app ready to be executed.
 
-Head back to the [Getting started](/docs/get_started)
+Head back to the [Getting started](/get_started)
```

Comment:
I think the correct link is `/docs/all/get_started`

Reply:
thanks, I was looking at the GitHub folder structure

I think the links won't work from GitHub though

Reply:
5a19708659f5f91470480ad9f8dfc9238c2ba85c

Reply:
Ticketed a related issue: https://github.com/ignitionrobotics/docs/issues/85